### PR TITLE
Style fixes to the running panel

### PR DESCRIPTION
**Pull request by ellisonbg**
General style improvements to the Running panels. Also partially addresses #1611

### CODE REVIEW

File: src/running/index.css [hunk 1]
```diff
@@ -5,6 +5,8 @@
 :root {
   --jp-private-running-button-height: 28px;
   --jp-private-running-button-width: 48px;
+  --jp-private-running-item-height: 24px;
+  --jp-prinate-running-shutdown-button-height: 20px;
```
Comment (afshin) [inline src/running/index.css]: `--jp-prinate-running-shutdown-button-height: 20px;`

Typo: `prinate` => `private`

File: src/running/index.css [hunk 2]
```diff
@@ -144,29 +136,19 @@
...
-  margin-right: 8px;
+  padding: 0px 4px;
+  margin: 2px 8px 2px 0px;
   color: var(--jp-inverse-ui-font-color0);
   background: var(--jp-warn-color1);
-  border-radius: 1px;
   cursor: pointer;
-  letter-spacing: .8px;
-  font-weight: 600;
+  height: var(--jp-prinate-running-shutdown-button-height);
```
Comment (afshin) [inline src/running/index.css]: Ditto re: `prinate` => `private`

File: src/running/index.css [hunk 3]
```diff
@@ -144,29 +136,19 @@
...
+  line-height: var(--jp-prinate-running-shutdown-button-height);
```
Comment (afshin) [inline src/running/index.css]: Ditto re: `prinate` => `private`

**Comment by ellisonbg** [comment]
@afshin ok review comments addressed!

**Comment by blink1073** [comment]
Looks good, thanks!

**Pull request merged**